### PR TITLE
Removed Facing: 0 and Health: 1

### PR DESCRIPTION
**Pull request by abcdefg30**
Split from #7261 as requested (https://github.com/OpenRA/OpenRA/pull/7261#issuecomment-69472628).

This PR removes "Facing: 0" on actors that don't use facings and "Health: 1" as it's default in all map.yaml files.

### CODE REVIEW

**Comment by Mailaender** [comment]
This is a followup of https://github.com/OpenRA/OpenRA/pull/7269 which fixed it in the legacy map import utility.

**Comment by Mailaender** [comment]
:+1:

File: mods/cnc/maps/gdi04a/map.yaml [hunk 1]
```diff
@@ -221,305 +221,229 @@ Actors:
...
 	HandOfNod: hand
 		Location: 30,52
 		Owner: Nod
-		Health: 1
-		Facing: 0
 	Hunter1: bggy
 		Location: 35,43
 		Owner: Nod
-		Health: 1
-		Facing: 192
```
Comment (obrakmann) [inline mods/cnc/maps/gdi04a/map.yaml]: here's one too many

**Comment by obrakmann** [comment]
:+1: when that one line has been restored.

Merging this should wait until #7261 has been merged, since both of these PRs will conflict pretty badly and this one is easier to recreate.

**Comment by Mailaender** [comment]
Actually the other way around. https://github.com/OpenRA/OpenRA/pull/7261 should remove the identical commit.

**Comment by obrakmann** [comment]
7261 is only 1 commit...

**Comment by Mailaender** [comment]
It should not include the `Health: 1` removals which are redundant to this pull request.

**Comment by abcdefg30** [comment]
Updated.

**Comment by abcdefg30** [comment]
> It should not include the  Health: 1  removals which are redundant to this pull request.

Removed them.

**Comment by Phrohdoh** [comment]
:+1:

File: mods/cnc/maps/gdi04b/map.yaml [hunk 2]
```diff
@@ -332,248 +332,189 @@ Actors:
...
 	Hunter1: bggy
 		Location: 27,37
 		Owner: Nod
-		Health: 1
-		Facing: 0
 	Hunter2: bggy
 		Location: 41,29
 		Owner: Nod
-		Health: 1
-		Facing: 128
```
Comment (obrakmann) [inline mods/cnc/maps/gdi04b/map.yaml]: this should be restored

File: mods/cnc/maps/gdi04b/map.yaml [hunk 3]
```diff
@@ -332,248 +332,189 @@ Actors:
...
 	Actor97: bggy
 		Location: 53,26
 		Owner: Nod
-		Health: 1
 		Facing: 128
 	tank: ltnk
 		Location: 8,17
 		Owner: Nod
-		Health: 1
-		Facing: 96
```
Comment (obrakmann) [inline mods/cnc/maps/gdi04b/map.yaml]: and this, too

**Comment by abcdefg30** [comment]
Updated.

**Pull request merged**